.github/workflows/python-app.yml

```
@@ -13,24 +13,64 @@ permissions:
       working-directory: ./backend
       run: make test-unit
+    
     - name: Run integration tests
       working-directory: ./backend
       run: make test-integration
+    
+    - name: Run tests with coverage
+      working-directory: ./backend
+      run: |
+        TESTING=true . .venv/bin/activate && python -m pytest tests/ --cov=tarsy --cov-report=html --cov-report=xml --cov-report=term-missing
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)